src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -91,7 +91,7 @@ public void testInvalidOption() throws Exception {
         try {
             final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
-            verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+            verify(checkConfig, getPath("InputSeparatorWrapAfter3904.java"), expected);
```

Comment:
@sagar-shah94 
Why you cannot rename the interfaces. You are free to rename any inner language structures. The main idea is to make it easy for future contributor to add new tests and understand your test. That is why it is better to split input files to shorten the context of the test.